backend/tests/DatabaseDuccScores.test.ts

```diff
@@ -0,0 +1,28 @@
+import { DatabaseDuccUtils } from '../database/DuccScores';
+import { IrcMessageProcessor } from '../irc/IrcMessageProcessor';
+import { JoinConfig } from '../irc/ircconnection';
+
+const mockJoinConfig: JoinConfig = {
+  channel: '#aboftytest',
+  user: 'lmraboft',
+  bufferTime: new Date(),
+};
+
+test('parse scores correctly based off of friend or killer', async () => {
+  const mockFriendScoresMsg =
+    ':gonzobot!gonzo@snoonet/bot/gonzobot PRIVMSG #aboftytest :(aboft) Duck friend scores in #aboftytest: a​bofty: 1,777 • a​boft: 13213 • Asmodean: 6,9,49';
+  const mockKillerScoresMsg =
+    ':gonzobot!gonzo@snoonet/bot/gonzobot PRIVMSG #aboftytest :(aboft) Duck killer scores in #aboftytest: a​smodean: 1 • a​boft: 13213 • Asmodean: 6,9,49';
+  const ircMessageProcessor = await IrcMessageProcessor.Instance(null!, mockJoinConfig);
+  const mockFriendScores = await ircMessageProcessor.parseMessage(mockFriendScoresMsg);
+  const mockKillerScores = await ircMessageProcessor.parseMessage(mockKillerScoresMsg);
+  await ircMessageProcessor.processIrcMessage(mockFriendScores);
+  await ircMessageProcessor.processIrcMessage(mockKillerScores);
+});
+
+test('insert or update ducc scores', async () => {
+  const friendScore = ['aboft: 1234', 'asmodean: 1,324,23'];
+  const killerScore = ['audron: 5,443', 'alexendoo: 44'];
+  await DatabaseDuccUtils.insertOrUpdateDuccScores(friendScore, 'friend');
+  await DatabaseDuccUtils.insertOrUpdateDuccScores(killerScore, 'killer');
+});
```

Comment:
These tests should have assertions, like [here](https://github.com/r-lmr/lmrdashboard/blob/lmrdashboard/backend/tests/DataBaseUserUtils.test.ts#L22). I see the last test case in that file doesn't have one either though.

Either return value or calledWith assertions I guess.

Reply:
we will make another PR to address the test and use this PR to fix the bug.